migrations/versions/6-phonebook.py

```diff
@@ -25,7 +25,19 @@ def upgrade() -> None:
         sa.Column("id", sa.String(), nullable=False),
         sa.Column("name", sa.String(), nullable=False),
         sa.Column("description", sa.String(), nullable=True),
-        sa.Column("kind", sa.String(), nullable=False),
+        sa.Column(
```

Comment:
Isn't it possible to use the enum define in type rather than hard code values twice?